📝 Update documentation (#81)
CloudNativeSDWAN/egress-watcher: README.md

### Supported Egress types

As of now, we support egress hosts defined as *ISTIO* `ServiceEntry` objects
or as IPs defined in the `Egress` fields of a *Kubernetes* `NetworkPolicy`,
and we reflect the changes we detect in them.

> The project is designed to accomodate different types defined by other

Reviewer: ```suggestion
The project is designed to accomodate different types of egress policies or external services defined by other
```

For a quick test of the project, you can try the `scripts/quickstart.sh` script
included in this repository that will guide you through deploying *Egress

> Watcher* iteratively with some default values and working with a `ServiceEntry`

Reviewer: ```suggestion
Watcher* iteratively with some default values and working with an Istio's `ServiceEntry`
```

```

> Try to deploy a `ServiceEntry` object you can use the provided example in

Reviewer: ```suggestion
Try to deploy a `ServiceEntry` object. You can use the provided example in
```

`artifacts/yamls/examples/istio`:

```bash
# In another shell terminal
kubectl create -f ./artifacts/yamls/examples/istio
```

> or a `NetworkPolicy`:

Reviewer: ```suggestion
For a `NetworkPolicy` you can do instead:
```
